pkg/services/ngalert/schedule/schedule.go

```diff
@@ -466,32 +477,39 @@ func (sch *schedule) ruleRoutine(grafanaCtx context.Context, key models.AlertRul
 		alerts := FromAlertStateToPostableAlerts(processedStates, sch.stateManager, sch.appURL)
 
 		if len(alerts.PostableAlerts) == 0 {
-			logger.Debug("no alerts to put in the notifier")
+			logger.Debug("no alerts to put in the notifier or to send to external Alertmanager(s)", "org", alertRule.OrgID)
```

Comment:
You do not need to include orgID in the message because it is already in the context. see line 455